frontend/src/components/SettingsPage.tsx

```diff
@@ -125,7 +127,7 @@ const SettingsPage = ({ user }: Props) => {
     
     setSaved(true)
     setTimeout(() => {
-      window.location.reload()
+      window.location.reload(false)
```

Comment:
location.reload has parameters (https://developer.mozilla.org/en-US/docs/Web/API/Location/reload#location.reload_has_no_parameter)

```suggestion
      window.location.reload()
```